src/libcsg/beadmotifalgorithms.cc

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2009-2020 The VOTCA Development Team (http://www.votca.org)
+ * Copyright 2009-2019 The VOTCA Development Team (http://www.votca.org)
```

Comment:
Huh?

Reply:
That's weird. I might have a old version of the git pre-commit hook in that repo.